src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeService.kt

```diff
@@ -92,7 +94,11 @@ class VilkårperiodeService(
             validerKanLeggeTilMålgruppeManuelt(behandling.stønadstype, vilkårperiode.type)
         }
         validerAktivitetsdager(vilkårPeriodeType = vilkårperiode.type, aktivitetsdager = vilkårperiode.aktivitetsdager)
```

Comment:
Denne trengs ikke lenger ettersom vi har
` require(aktivitetsdager in 1..5) { "Aktivitetsdager må være mellom 1 og 5" }`
på `FaktaAktivitetTilsynBarn`

Reply:
Denne funksjonen skal slettes